main/Accordion/AccordionItem.tsx

```diff
@@ -74,12 +81,21 @@ const AccordionItem: FC<Props> = (props) => {
     toggleElement = <StyledIcon name="chevron-down-light" />,
     dropDownAnimValueList,
     sumOfPrecedingTranslateY,
-    renderTitle = (title) => <StyledTitle>{title}</StyledTitle>,
-    renderBody = (body) => <StyledItem>{body}</StyledItem>,
+    renderTitle = (title) => (
+      <StyledTitle style={{color: textContrast}}>{title}</StyledTitle>
+    ),
+    renderBody = (body) => (
+      <StyledItem style={{color: text}}>{body}</StyledItem>
```

Comment:
위에서 언급한 부분의 코드
color style을 스타일 컴포넌트에서 분리

Reply:
Since default theme is not applied to context of `emtion`, default theme is not visible to `Styled`. So to support it, you should manually give style props. Or just remove `Styled` component. 

I'm looking  for way to support default theme for context of `emotion` too, but not done yet.

For further understanding, see code in `emotion`, or [test code relate to this](https://github.com/dooboolab/dooboo-ui/blob/4884e59ef79e5cc02d2abc920b5442023830bd84/packages/Theme/ThemeProvider.test.tsx#L137-L165), or feel free to ask me.

Reply:
I think that is the best way as of yet. thank you kindly!